ui/src/app/components/object-explorer/object-explorer.component.html

```diff
@@ -28,29 +28,32 @@
           <ng-container matColumnDef="status">
             <th mat-header-cell *matHeaderCellDef>Status</th>
             <td mat-cell *matCellDef="let data">
-              <mat-icon
-                *ngIf="data.status === 'GREEN' || data.status === 'GRAY'"
-                class="icon success"
-                matTooltip="Can be converted automatically"
-                matTooltipPosition="above"
-              >
-                check_circle
-              </mat-icon>
+              <svg  class="icon success"
```

Comment:
Lets keep the colour as black for all the icons

Reply:
Updated